src/resources/Correction/BenefitCompanyStatementResource.ts

```diff
@@ -21,7 +23,8 @@ export const BenefitCompanyStatementResource: any = {
       CorrectionTypes.CORRECT_NAME
     ],
     orgPersonInfo: {
-      orgPersonLabel: 'Directors'
+      orgPersonLabel: 'Directors',
+      subtitle: null // not used
```

Comment:
Now that this resource is typed (using interface), this property was missing, and I didn't want to make it optional.